app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -92,33 +104,145 @@ function ReviewIssuesStep({
+                  )}
+            </p>
+          )}
+        </Warning>
+      );
+    }
+    if (inPersonCtaVariantActive === 'in_person_variant_b') {
+      trackEvent('IdV: IPP CTA Variant B');
+      return (
+        <Warning
+          heading={t('errors.doc_auth.throttled_heading')}
+          actionText={t('idv.failure.button.warning_variant')}
+          actionOnClick={onWarningPageDismissed}
+          altActionText={t('in_person_proofing.body.cta.button_variant')}
+          altActionOnClick={onInPersonSelected}
+          altHref="#location"
+          location="doc_auth_review_issues"
+          remainingAttempts={remainingAttempts}
+          troubleshootingOptions={
+            <DocumentCaptureTroubleshootingOptions
+              location="post_submission_warning"
+              showAlternativeProofingOptions={false}
+              heading={t('components.troubleshooting_options.ipp_heading')}
+            />
+          }
+        >
+          {!!unknownFieldErrors &&
+            unknownFieldErrors
+              .filter((error) => !['front', 'back'].includes(error.field!))
+              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
+
+          {remainingAttempts <= DISPLAY_ATTEMPTS && (
+            <p>
+              {remainingAttempts === 1
+                ? formatWithStrongNoWrap(t('idv.failure.attempts.one_variant_b_html'))
+                : formatWithStrongNoWrap(
+                    t('idv.failure.attempts.other_variant_b_html', { count: remainingAttempts }),
+                  )}
+            </p>
+          )}
+          <p>{t('in_person_proofing.body.cta.prompt_detail_b')}</p>
+        </Warning>
+      );
+    }
+    if (inPersonCtaVariantActive === 'in_person_variant_c') {
+      trackEvent('IdV: IPP CTA Variant C');
+      return (
+        <Warning
+          heading={t('errors.doc_auth.throttled_heading')}
+          actionText={t('idv.failure.button.warning')}
+          actionOnClick={onWarningPageDismissed}
+          location="doc_auth_review_issues"
+          remainingAttempts={remainingAttempts}
+          troubleshootingOptions={
+            <DocumentCaptureTroubleshootingOptions
+              location="post_submission_warning"
+              showAlternativeProofingOptions={false}
+              heading={t('components.troubleshooting_options.ipp_heading')}
+            />
+          }
+        >
+          {!!unknownFieldErrors &&
+            unknownFieldErrors
+              .filter((error) => !['front', 'back'].includes(error.field!))
+              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
+
+          {remainingAttempts <= DISPLAY_ATTEMPTS && (
+            <p>
+              <strong>
+                {remainingAttempts === 1
+                  ? t('idv.failure.attempts.one')
+                  : t('idv.failure.attempts.other', { count: remainingAttempts })}
+              </strong>
+            </p>
+          )}
+        </Warning>
```

Comment:
This is a lot to copy. I would suggest aggregating the shared props and components into a few variables and reusing them among the different variants.

Reply:
trackEvent concern is handled in latest commit. 

DRYing up the variants and original RIDP flow may not be worthwhile, depending upon the anticipated lifespan of this particular AB test. If this is expected to run for longer, the suggested refactor will take place as part of a separate ticket. Cc: @tomas-nava